src/lib/isMobilePhone.js

```diff
@@ -96,11 +98,15 @@ const phones = {
   'fr-BJ': /^(\+229)\d{8}$/,
   'fr-CM': /^(\+?237)6[0-9]{8}$/,
   'fr-FR': /^(\+?33|0)[67]\d{8}$/,
+  'fr-GA': /^(\+241)(01(40|44|45|46|47|48|50|55|56|58|59|60|62|65|66|67|69|70|71|72|73|74|75|76|77|78|79|82|83|86|90|92|93|96|98)\d{4}|(1(54))\d{4}|(01(420)\d{3})|(01(424)\d{3}))$/,
   'fr-GF': /^(\+?594|0|00594)[67]\d{8}$/,
   'fr-GP': /^(\+?590|0|00590)[67]\d{8}$/,
   'fr-MQ': /^(\+?596|0|00596)[67]\d{8}$/,
+  'fr-ML': /^(\+223)((202|213|214|215|216|218|219|442)\d{5}|(2020|2021|2022|2023|2024|2025|2028|2029|2072|2077|2078|2079|2126|2127)\d{4})$/,
   'fr-PF': /^(\+?689)?8[789]\d{6}$/,
   'fr-RE': /^(\+?262|0|00262)[67]\d{8}$/,
+  'fr-TD': /^(\+235)(22)(50|51|52|54|68|69)\d{2}$/,
+  'fr-TG': /^(\+228)(22|23|24|25|26|27)\d{6}$/,
```

Comment:
```suggestion
  'fr-TG': /^(\+?228|0)[79]\d{7}$/,
```
https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000D10003PDFE.pdf